### ISSUE TITLE
loss function and LEM IR

### ISSUE THREAD

**Issue by handle787** [ThomasHaubner/e2e_dnn_ad_control_for_lin_aec#2]
Hello, thank you very much for open-sourcing your work. I am currently reproducing the work in the paper. I would like to ask you how the length and coefficient of LEM IR are determined in the paper. And when TD-ERLE is used as the loss function, what is the approximate loss when the training converges. Thank you!
